src/hpp/corbaserver/client.py

```diff
@@ -132,10 +132,10 @@ def _initOrb(self, url, host=None, port=None):
                     self.initWithNameService(url + "/NameService")
         else:
             urlNameService = _getIIOPurl(
-                service="NameService", host=host, port=port if port else 2809
+                service="NameService", host=host, port=port, default_port=2809
             )
             urlHppTools = _getIIOPurl(
-                service="hpp-corbaserver", host=host, port=port if port else 13331
+                service="hpp-corbaserver", host=host, port=port, default_port=13331
```

Comment:
```suggestion
                service="hpp-corbaserver", host=host, port=port if port else os.getenv("HPP_PORT", 13331)
```

maybe ? Otherwise I don't get the different use cases between this 13331 or the one you introduced in tools.py

In other words: If I ever want to change the default from beeing 13331 to 13332, should I change this `default_port=13331` here or the `port = os.getenv("HPP_PORT", 13331)` in tools.py ? and why would we have the 2 ?

Reply:
I defined defaultPort in class Client and use it in tools.py. Thank you for the remark.